OamLog.yaml

```diff
@@ -519,33 +581,33 @@ paths:
                       type: object
```

Comment:
line 512 : PromptForBequeathingDataCausesRObeingRequestedToStopNotificationsToOldRelease :: subscriber-release-number :: description :
Can we update the release-number description to be pointing to the http-server/release-number instead of OldRelease/release-number, because OldRelease will contain information of already outdated release of this application. And our intention is to request RO to stop notification to this application (which is expected to be deregistered - http-server)

Reply:
Changed as proposed.

Reply:
Thanks for updating the path.

Also, the first line of description shall be changed to "Own release number "

Reply:
Changed as proposed.